setup.sh

```diff
@@ -1,44 +1,39 @@
 #!/usr/bin/env bash
 set -e
 
-source ${BASH_SOURCE%/*}/vault_auth.sh
+HERE=${BASH_SOURCE%/*}
+source ${HERE}/vault_auth.sh
 
 echo "Installing Pip and required Python packages"
 apt-get install python3-pip -y > /dev/null
-pip3 install -q -r requirements.txt
+pip3 install -q -r ${HERE}/requirements.txt
 
 echo "-------------------------------------------"
 echo "Installing bb8 user and group"
 
 if ! id -u bb8 > /dev/null 2>&1; then
-    useradd bb8
+    useradd bb8 -d /usr/lib/bb8
```

Comment:
is `usr/lib/bb8` a better home dir than `var/log/bb8`?

Reply:
Sorry this was a typo!  What you had was correct, what I added was wrong